tls/s2n_resume.h

```diff
@@ -106,7 +106,7 @@ int s2n_connection_add_new_tickets_to_send(struct s2n_connection *conn, uint8_t
 int s2n_connection_set_server_keying_material_lifetime(struct s2n_connection *conn, uint32_t lifetime_in_secs);
 
 struct s2n_session_ticket;
-typedef int (*s2n_session_ticket_fn)(struct s2n_connection *conn, struct s2n_session_ticket *ticket);
+typedef int (*s2n_session_ticket_fn)(void *ctx, struct s2n_connection *conn, struct s2n_session_ticket *ticket);
```

Comment:
Nit: To match our other APIs, I don't think the context is ever the first argument. It's usually second or last.

Reply:
https://github.com/aws/s2n-tls/blob/main/bin/common.h#L52 ? We have a standard?